tensorflow-core/tensorflow-core-generator/src/main/java/org/tensorflow/processor/operator/OperatorProcessor.java

```diff
@@ -537,6 +538,18 @@ private static TypeSpec buildTopClass(OpsSpec spec) {
                 T_SCOPE)
             .build());
 
+    opsBuilder.addMethod(
+            MethodSpec.methodBuilder("withDevice")
+                    .addModifiers(Modifier.PUBLIC)
+                    .addParameter(T_DEVICE_SPEC, "deviceSpec")
+                    .returns(T_OPS)
+                    .addStatement("return new Ops(scope.withDevice(deviceSpec))")
+                    .addJavadoc(
+                            "Returns an API that uses the provided DeviceSpec for an op.\n\n"
+                                    + "@see {@link $T#withDevice(DeviceSpec)}\n",
+                            T_SCOPE)
+                    .build());
+
```

Comment:
The `Ops` class should now have this new method in it but it does not appear in the version you've committed.

Also, I would rephrase its documentation for something like this:

"Returns an API that places the created operations on the device(s) matching the provided spec. @see..."